packages/api/portfolioDrafts/submit/consumePortfolioDraftSubmitQueue.ts

```diff
@@ -0,0 +1,20 @@
+import { SQSEvent } from "aws-lambda";
+import { sfnClient } from "../../utils/aws-sdk/stepFunctions";
+
+const SFN_ARN = process.env.SFN_ARN ?? "";
+
+/**
+ * Receives SQS Events and starts the State Machine Execution
+ *
+ * @param event - The SQS Event, triggered when a message is sent to the queue
+ */
+export async function handler(event: SQSEvent): Promise<void> {
+  const records = event.Records.map((record) => record.body);
+  console.log("Number of Records: " + records.length);
+  console.log("Sent Record: " + records[0]);
+  const result = await sfnClient.startExecution({
+    input: records[0],
+    stateMachineArn: SFN_ARN,
+  });
+  console.log("Response: " + JSON.stringify(result));
```

Comment:
Should this be in a for loop?

Reply:
At the moment the `batchSize` is set to one to prevent batch failures from SQS, but yes it probably should be in a for loop should we change the `batchSize`.  Adding for loop.

Reply:
Yeah, since it's a list anyway (even a list of one item), we may as well write the function so that it supports multiple items.